apps/admin-gui/src/app/vos/pages/vo-select-page/vo-select-page.component.html

```diff
@@ -3,10 +3,10 @@
   </perun-web-apps-back-button>
   <h1 class="page-title">{{'VO_MANAGEMENT.TITLE' | translate}}</h1>
   <perun-web-apps-refresh-button (refresh)="refreshTable()" *ngIf="!loading"></perun-web-apps-refresh-button>
-  <button mat-flat-button color="accent" class="mr-2" (click)="onCreateVo()" *ngIf="!loading && isVoAdmin">
+  <button mat-flat-button color="accent" class="mr-2" (click)="onCreateVo()" *ngIf="!loading && isAuthorized">
     {{'VO_MANAGEMENT.CREATE_VO' | translate}}
   </button>
-  <button mat-flat-button color="warn" (click)="onRemoveVo()" *ngIf="!loading && isVoAdmin" [disabled]="selection.selected.length === 0">
+  <button mat-flat-button color="warn" (click)="onRemoveVo()" *ngIf="!loading && isAuthorized" [disabled]="selection.selected.length === 0">
```

Comment:
please use for this other policy than createVo (somethink like deleteVo_Vo_policy would be better)